app/src/main/java/covid/trace/morocco/LocaleHelper.kt

```diff
@@ -0,0 +1,92 @@
+package covid.trace.morocco
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.annotation.StringDef
+import io.github.inflationx.calligraphy3.CalligraphyConfig
+import io.github.inflationx.calligraphy3.CalligraphyInterceptor
+import io.github.inflationx.viewpump.ViewPump
+import java.lang.annotation.Retention
+import java.lang.annotation.RetentionPolicy
+import java.util.*
+
+object LocaleHelper {
+
+    private const val SHARED_PREF_NAME = "covid_19"
+    private const val PATH_FONT_FR = "fonts/Barlow-Regular.ttf"
+    private const val PATH_FONT_AR = "fonts/Almarai-Regular.ttf"
+    private const val LANGUAGE_PREF_KEY = "language_pref_key"
+    const val FRENCH = "fr"
+    const val ARABIC = "ar"
+
+    private var sharedPreferences: SharedPreferences? = null
+    var currentLocale: String? = null
```

Comment:
non optional `currentLocale` here

Reply:
currentLocale can be null if a class does not call the method, but it should be private